scrapy/contrib/exporter/__init__.py

```diff
@@ -154,6 +154,7 @@ def _export_xml_field(self, name, serialized_value):
     if sys.version_info[:3] >= (2, 7, 4):
         def _xg_characters(self, serialized_value):
             if not isinstance(serialized_value, unicode):
+                serialized_value = str(serialized_value)
```

Comment:
From the source code it seems they'll work fine - _export_xml_field method calls itself recursively, so _xg_characters method shouldn't receive containers (lists or dicts). 

A test for this would be nice though.